x/btcstkconsumer/types/chain_register.go

```diff
@@ -0,0 +1,49 @@
+package types
+
+import (
+	"fmt"
+)
+
+func (m *MsgRegisterConsumer) ValidateBasic() error {
+	if len(m.ConsumerId) == 0 {
+		return fmt.Errorf("ConsumerId must be non-empty")
+	}
+	if len(m.ConsumerName) == 0 {
+		return fmt.Errorf("ConsumerName must be non-empty")
+	}
+	if len(m.ConsumerDescription) == 0 {
```

Comment:
why do we allow not specify the description in CLI? 
https://github.com/babylonlabs-io/babylon/blob/ce31233233d496868bd1c83794c30ec3456c0b16/x/btcstkconsumer/client/cli/tx.go#L56

Reply:
This is the default value, and the CLI input will be taken [here](https://github.com/babylonlabs-io/babylon/blob/1a0f10f4f7918a5b792a42dcc1b18c6b13c17d73/x/btcstkconsumer/client/cli/tx.go#L58-L61)

Reply:
Good point, let's enforce this in CLI